src/test/java/teammates/test/pageobjects/AdminEmailPage.java

```diff
@@ -20,6 +33,18 @@ public void inputRecipient(String recipient) {
         recipientBox.sendKeys(recipient);
     }
 
+    /**
+     * Makes the groupReceiverListUploadBox visible, uploads file and makes it invisible again.
+     * @param fileName to be uploaded
+     */
+    public void inputGroupRecipient(String fileName) {
+        executeScript("arguments[0].style.display = 'inline'", groupReceiverListUploadBox);
```

Comment:
A quick question. Is it necessary to make the groupReceiverListUploadBox visible in order to upload file?